api/src/main/scala/hmda/api/http/HttpApi.scala

```diff
@@ -25,6 +41,37 @@ trait HttpApi extends HmdaApiProtocol {
           ToResponseMarshallable(status)
         }
       }
-    }
+    } ~
+      path("upload") {
+        post {
+          val processingActor = system.actorOf(ProcessingActor.props)
+          entity(as[Multipart.FormData]) { formData =>
+            val uploaded: Future[Done] = formData.parts.mapAsync(1) {
+              //TODO: check Content-Type type as well?
+              case b: BodyPart if b.filename.exists(_.endsWith(".csv")) =>
```

Comment:
In the [2017 HMDA File Specification](http://www.consumerfinance.gov/hmda/static/for-filers/2017/2017-HMDA-File-Specifications.pdf), we say:

>The file will be a text file with a `.txt` file format extension.